src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -127,18 +127,10 @@ object DepositHandler {
     }
   }
 
-  def checkBaseDir()(implicit id: String, baseDir: File): Try[Unit] = {
-    if (!baseDir.exists()) {
-      log.error(s"[$id] Bag store base directory ${baseDir.getAbsolutePath} doesn't exist")
-      Failure(FailedDepositException(id, s"Bag store base directory ${baseDir.getAbsolutePath} doesn't exist"))
-    }
-    else if (!baseDir.canRead) {
-      log.error(s"[$id] Bag store base directory ${baseDir.getAbsolutePath} is not readable")
-      Failure(FailedDepositException(id, s"Bag store base directory ${baseDir.getAbsolutePath} is not readable"))
-    }
-    else {
-      Success(())
-    }
+  def checkBagStoreBaseDir()(implicit id: String, baseDir: File): Try[Unit] = {
```

Comment:
The reason to add the logging was that I didn't find any logging anywhere. Most likely I just didn't look in the right place...